src/tools/Altinn.Authorization.CLI/src/Altinn.Authorization.CLI/Database/DatabaseBootstrapper.cs

```diff
@@ -0,0 +1,411 @@
+{
+    /// <summary>
+    /// Console for displaying messages and progress.
+    /// </summary>
+    public IAnsiConsole Console { get; } = console;
+
+    /// <summary>
+    /// Configuration options for the bootstrapper.
+    /// </summary>
+    public DatabaseBootstrapperOptions Options { get; } = options;
+
+    /// <summary>
+    /// Runs the bootstrapping process to set up PostgreSQL resources.
+    /// </summary>
+    public async Task<int> Run()
+    {
+        try
+        {
+            var token = new DefaultAzureCredential();
+            var postgresArm = new ArmClient(token, Options.ServerSubscription);
+            var keyVaultArm = new ArmClient(token, Options.KeyVaultSubscription);
+
+            if (!Options.ConfigFile.Exists)
+            {
+                throw new ArgumentException($"file {Options.ConfigFile.FullName} does not exist");
+            }
+
+            var content = await File.ReadAllTextAsync(Options.ConfigFile.FullName);
+            var config = JsonSerializer.Deserialize<AppsConfig>(content);
+
+            var postgresResource = await GetPostgresFlexibleServerResource(postgresArm);
+            var keyVaultResource = await GetArmKeyVaultResource(keyVaultArm);
+            var connectionString = await CreateAdminConnectionString(token, postgresResource);
+            var secretClient = new SecretClient(keyVaultResource.Data.Properties.VaultUri, token);
+
+            await using var conn = new NpgsqlConnection(connectionString.ToString());
+            await conn.OpenAsync();
+
+            var migratorUser = await CreateDatabaseRole(conn, secretClient, $"{config.Database.Prefix}_migrator", connectionString.Username);
+            var appUser = await CreateDatabaseRole(conn, secretClient, $"{config.Database.Prefix}_app", connectionString.Username);
+            await CreateDatabase(conn, config.Database.Name);
+            await GrantDatabasePrivileges(conn, config.Database.Name, migratorUser.RoleName, "CREATE, CONNECT");
+            await GrantDatabasePrivileges(conn, config.Database.Name, appUser.RoleName, "CONNECT");
+
+            foreach (var (schemaName, schemaCfg) in config.Database.Schemas)
+            {
+                await CreateDatabaseSchema(conn, migratorUser, schemaName, schemaCfg);
+            }
+
+            await StoreConnectionString(config, migratorUser, postgresResource.Data.FullyQualifiedDomainName, secretClient);
+            await StoreConnectionString(config, appUser, postgresResource.Data.FullyQualifiedDomainName, secretClient);
+        }
+        catch (Exception ex)
+        {
+            Console.WriteException(ex);
+            return 1;
+        }
+
+        return 0;
+    }
+
+    private async Task StoreConnectionString(AppsConfig config, Result user, string serverUrl, SecretClient secretClient, CancellationToken cancellationToken = default)
+    {
+        var connectionString = new NpgsqlConnectionStringBuilder()
```

Comment:
Hvordan håndteres connection pool size pr. miljø og pr. pod nå?

For dagens database har vi gjort en manuell fordeling av connections og lagt dette i connectionstring pr. miljø config.

Reply:
i så fall kan dette bakes inn i conf.json? tanker?

Reply:
Eller dette kan også konfigureres i app configuration i azure